state/migration_internal_test.go

```diff
@@ -204,6 +204,11 @@ func (s *MigrationSuite) TestKnownCollections(c *gc.C) {
 		relationNetworksC,
 		firewallRulesC,
 		dockerResourcesC,
+		// Raft leases - TODO
```

Comment:
// TODO(raft-leases)

(or whatever we use for consistency)

Reply:
Good spot!